generators/app/package.json

```diff
@@ -2,18 +2,22 @@
   "name": "polymer-starter-kit-custom-build",
   "version": "0.0.0",
   "description": "A starting point for Polymer 1.0 custom build apps",
-  "license": "http://polymer.github.io/LICENSE.txt",
+  "license": "BSD-3-Clause",
```

Comment:
@FredKSchott I based this on other Polymer projects, https://github.com/Polymer/web-component-tester/blob/master/package.json#L21